docs/DeveloperGuide.md

```diff
@@ -235,6 +235,33 @@ Pros | Cons
 -----|-----
 \+ More well-known among developers | - Less technical users may not know how to open a .json file.
 
+### 3.2 Listing all student groups
+
+#### 3.2.1 Implementation
+The listing all student groups feature is facilitated by `ListGroupCommand`. It extends `Command` and overrides `Command#execute()` to list all student groups.
+
+Given below is a step-by-step usage scenario and how the listing all student groups feature works:
+
+1. The user launches the application and types `list-group` into the input box.
+2. The `UI` handles the input and calls `LogicManager#execute()` to execute it.
+3. The `AddressBookParser#parseCommand()` is called to parse the input and return a `ListGroupCommand`.
```

Comment:
Should it be `......is called... and **returns**...`?

Reply:
Noted, thanks.